API/src/utils/Config/index.js

```diff
@@ -19,9 +19,9 @@ module.exports = {
     host: process.env.DATABASE_HOST || `localhost`,
     port: process.env.DATABASE_PORT || 5432,
     username: process.env.DATABASE_USERNAME || `postgres`,
-    password: process.env.DATABASE_PASSWORD || `paul`,
-    name: process.env.DATABASE_NAME || `risk`,
+    password: process.env.DATABASE_PASSWORD || `Bodensjr`,
+    name: process.env.DATABASE_NAME || `Ocat`,
```

Comment:
```suggestion
    password: process.env.DATABASE_PASSWORD || ``,
    name: process.env.DATABASE_NAME || ``,
```
Same thing here.  Add these in your config.json so you don't have to expose the password to the cloud